src/components/Tools/UsefulLinks.vue

```diff
@@ -28,6 +28,10 @@ export default defineComponent({
           title: 'Class Roster',
           link: 'https://classes.cornell.edu/browse/',
         },
+        {
+          title: 'Academic Calendar',
+          link: 'https://registrar.cornell.edu/academic-calendar',
```

Comment:
trailing slash for consistency?

```suggestion
          link: 'https://registrar.cornell.edu/academic-calendar/',
```